README.md

```diff
@@ -1 +1,213 @@
-# animated_book_list
\ No newline at end of file
+![banner](https://i.imgur.com/ppKK1L0.jpeg)
+
+# Flutter Animated Book Widget
+
+![build](https://github.com/the-d-velopers/animated_book_list/actions/workflows/code_metrics.yml/badge.svg?branch=master)
```

Comment:
The branch name should be `main`

Reply:
I need confirmation, which github badge URL did we use?

* with branch parameter:

![Captura de pantalla 2024-01-29 a la(s) 11 01 57](https://github.com/the-d-velopers/animated_book_widget/assets/70081671/f35192ed-be0d-4eb4-bbe8-55ca85824730)

* without:
![Captura de pantalla 2024-01-29 a la(s) 11 05 44](https://github.com/the-d-velopers/animated_book_widget/assets/70081671/fb848c66-efbe-46ed-8dee-104c81c01340)